### PR TITLE
PP-508/fees-from-server

### PR DESCRIPTION
**Pull request by franciscotobar**
## What

-   Consume the endpoint from the Relay Server for gas estimation

## Why

-  Simplify the gas estimation
- Centralize the source of the gas estimation

## Refs
- [Jira Issue](https://rsklabs.atlassian.net/browse/PP-508)
- https://github.com/rsksmart/rif-relay-client/pull/36
- https://github.com/rsksmart/rif-relay-sdk/pull/31

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@openeth/truffle-typings": "0.0.6",
-    "@rsksmart/rif-relay-sdk": "1.0.0-alpha.0",
+    "@rsksmart/rif-relay-sdk": "github:rsksmart/rif-relay-sdk#PP-508/fees-from-server",
```
Comment (antomor) [inline package.json]: We need to update it after merging the related PRs

File: src/modals/Execute.tsx [hunk 2]
```diff
@@ -77,6 +84,19 @@ function Execute() {
...
+    const close = () => {
+        dispatch({ type: 'set_modals', modal: { execute: false } });
+        setResults('');
+        setExecute({
+            check: false,
+            show: false,
+            address: '',
+            value: '',
+            function: '',
+            fees: ''
```
Comment (antomor) [inline src/modals/Execute.tsx]: Could we store this into an `initialState` variable?

**Review by antomor — CHANGES_REQUESTED** [review]
I haven't tested the dapp yet, and I'm going to do it after merging the other PRs, so I can test the whole system.

File: src/modals/Execute.tsx (hunk 2 above)
Comment (franciscotobar) [reply src/modals/Execute.tsx]: updated

**Review by antomor — APPROVED** [review]
LGTM

**Pull request merged**